src/components/common/TmBalance.vue

```diff
@@ -248,6 +250,30 @@ export default {
       },
       /* istanbul ignore next */
       update(data) {
+        if (!this.sentToGA) {
```

Comment:
why only once?

Reply:
Otherwise, it will send it every 5 seconds.

Reply:
ah right, not a problem though I guess ^^